index.html

```diff
@@ -38,37 +44,39 @@ <h2 class="contentLink">LET'S CONNECT</h2>
 
 <section class="grid-container">
     <div class="grid-item">
-        <img id="header-img" src="./img/portfolio01.jpg" >
-            
-        <h2>Tonic</h2>
-            
-        <section class="details">
-        <ul>
-            <li>
-                <div>CANOPY</div>
-                <img src="./img/counter.png">
-                <div>Back End Dev</div>
-                <img src="./img/counter.png">
-                <div>2015</div>
-            </li>
-        </ul>
-        </section>
-
-        <p class="worksText">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-        <section class="worksTag">
+        <img id="header-img-l" src="./img/portfolio01.jpg" >
+        
+        <div class="postLeft">
+            <h2>Tonic</h2>
+                
+            <section class="details">
```

Comment:
I have seen the following things that need to be redesigned according to the Desktop view.
1. `font-size` of all elements
2. Alignment of all elements according to Desktop view
3. The width and height of work cards should have dimensions according to Desktop View.

| Your Design   | FIGMA Design  |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/61361037/221363523-803356cf-d06f-4530-bf88-799a3bcac1db.png)| ![image](https://user-images.githubusercontent.com/61361037/221363437-6435c3ce-3dfd-4938-bb2d-769f13a84655.png)|